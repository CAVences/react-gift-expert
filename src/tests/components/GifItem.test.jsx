
import { render } from "@testing-library/react";
import { GifItem } from "../../components/GifItem";

describe('Pruebas en <GifItem></GifItem>', () => {
    const title = 'pitochu';
    const url = 'https://pikachu.com/pitochu.jpg';

    test('match con snpapshot', () => {
        const {container} = render(<GifItem title={title} url={url}/>);
        expect(container).toMatchSnapshot();
    })
})
