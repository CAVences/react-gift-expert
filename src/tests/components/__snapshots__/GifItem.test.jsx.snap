// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <GifItem></GifItem> match con snpapshot 1`] = `
<div>
  <div
    class="card"
  >
    <img
      alt="pitochu"
      src="https://pikachu.com/pitochu.jpg"
    />
    <p>
      pitochu
    </p>
  </div>
</div>
`;
